app/views/protocols/index/_protocol_preview_modal_body.html.erb

```diff
@@ -68,88 +64,22 @@
-                        <%= hidden_field(table, :contents, value: table.contents_utf_8, class: "hot-contents") %>
-                        <div data-role="step-hot-table" class="step-result-hot-table"></div>
-                      </div>
-                    <% end %>
-                  </div>
-                <% end %>
-
-                <% assets = ordered_assets(step) %>
-                <% unless assets.blank? then %>
-                  <hr>
-                  <div class="col-xs-12">
-                    <strong><%= t("protocols.steps.files") %></strong>
-                    <ul>
-                      <% assets.each do |asset| %>
-                        <li>
-                          <%= render partial: "shared/asset_link", locals: { asset: asset, display_image_tag: true }, formats: :html %>
-                        </li>
-                      <% end %>
-                    </ul>
-                  </div>
-                <% end %>
-
-                <% unless step.checklists.blank? then %>
-                  <div class="col-xs-12">
-                    <% step.checklists.asc.each do |checklist| %>
-                      <strong><%= checklist.name %></strong>
-                      <% if checklist.checklist_items.empty? %>
-                        </br>
-                        <%= t("protocols.steps.empty_checklist") %>
-                        </br>
-                      <% else %>
-                        <% ordered_checklist_items(checklist).each do |checklist_item| %>
-                          <div class="checkbox">
-                            <label>
-                              <% if protocol.in_module? %>
-                                <input type="checkbox" value="" <%= "checked" if checklist_item.checked? %> disabled="disabled"/>
-                              <% else %>
-                                <input type="checkbox" value="" disabled="disabled" />
-                              <% end %>
-                              <%= checklist_item.text %>
-                            </label>
-                          </div>
-                        <% end %>
-                      <% end %>
-                    <% end %>
-                  </div>
-                <% end %>
-
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
+     <%= render partial: "steps/step.html.erb", locals: { step: step, preview: true } %>
     <% end %>
   </div>
 </div>
+
+<%= javascript_include_tag "handsontable.full.min" %>
+<%= javascript_include_tag "lodash" %>
+<%= javascript_include_tag "numeral" %>
+<%= javascript_include_tag "numeric" %>
+<%= javascript_include_tag "md5" %>
+<%= javascript_include_tag "jstat" %>
+<%= javascript_include_tag "formula" %>
+<%= javascript_include_tag "parser" %>
+<%= javascript_include_tag "ruleJS" %>
+<%= javascript_include_tag "handsontable.formula" %>
+<%= javascript_include_tag "big.min" %>
+<%= stylesheet_link_tag "handsontable.formula" %>
```

Comment:
Would it make sense to DRY up this as well (something like `_formuls_dependencies.html.erb`? It appears on couple of places as well.

Reply:
DRYed